frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -87,10 +145,10 @@ const NavMenu = () => {
             </Menu.Target>
 
             <Menu.Dropdown>
-                {isMobile && <Menu.Item><StyledLink to="/studies">Studies</StyledLink></Menu.Item>}
+                {isMobile && <Menu.Item><Link to="/study">Dashboard</Link></Menu.Item>}
```

Comment:
Not styled as per design, achievemens are also missing

Reply:
Achievements is being done in a different ticket

Reply:
So this one is needed to be done in the other ticket